docs/source/release_notes.rst

```diff
@@ -20,8 +28,8 @@ Future Release
         * Add unit test for dropping empty data slices (:pr:`280`)
         * Add auto approve workflow for dependency updates (:pr:`281`)
 
-    | Thanks to the following people for contributing to this release:
-    | :user:`gsheni`, :user:`jeff-hernandez`, :user:`thehomebrewnerd`
+    Thanks to the following people for contributing to this release:
+    :user:`gsheni`, :user:`jeff-hernandez`, :user:`thehomebrewnerd`
```

Comment:
Can we keep on separate lines?

![Screen Shot 2022-01-20 at 12 27 21 PM](https://user-images.githubusercontent.com/12969559/150399417-ed3deeb9-e922-47de-ab69-4a0c349f8c5a.png)

Reply:
Or we can also format the other entries -- main thing is just using a consistent format

Reply:
Yeah, I changed to match what we have been doing in other repos, but I didn't catch the other entries were like this too. I'll change this back for now to use separate lines to be consistent.